core/src/Platforms/android/LegacyCachePersistance.cs

```diff
@@ -50,8 +50,10 @@ byte[] ILegacyCachePersistance.LoadCache()
             }
             catch (Exception ex)
             {
-                CoreLoggerBase.Default.Warning("Failed to load cache: " + ex.Message);
-                CoreLoggerBase.Default.ErrorPii(ex);
+                string msg = "Failed to load cache: ";
```

Comment:
>Failed to load cache: [](start = 30, length = 21)

An error occurred while reading the adal cache?

Reply:
done.